XSUP-30439: TAXII 2 Feed Fix Fetch Indicators V2 (#32143)
demisto/content: Packs/FeedTAXII/ReleaseNotes/1_2_3.md

#### Integrations

##### TAXII 2 Feed
- Updated the Docker image to: *demisto/taxii2:1.0.0.84787*.

> - Fixed an issue where a connectivity issue occurs during **fetch-indicators** command, the digested indicators are not returned.

Reviewer: ```suggestion
- Added support for returning digested indicators in cases of connectivity issues during the ***fetch-indicators*** command.
```
